src/prefect/hello_world.py

```diff
@@ -0,0 +1,19 @@
+"""
+A very basic 'hello-world' flow for getting started quickly
+
+Examples:
+    > prefect register --project default -m prefect.hello_world
+    > prefect run -m prefect.hello_world --no-agent
```

Comment:
I may exclude this example until this feature is ready

Reply:
+1 would vote to exclude until this is ready;